pkg/api/v1/clients/common/common.go

```diff
@@ -58,14 +60,23 @@ func KubeResourceWatch(cache cache.Cache, listFunc ResourceListFunc, namespace s
 		defer close(errs)
 
 		// watch should open up with an initial read
+		timer := time.NewTicker(time.Second)
```

Comment:
can we make this configurable? 1s seems kind of slow for some use cases

also, what do you think of a feature flag so it can be disabled? (in the kube factory)

Reply:
i can't imagine 1s to be slow for any use case... what use case do you have in mind?
same goes for disabling this - i'm not sure i see a clear benefit that mertis this.. can you elaborate more?

Reply:
on second thought i think it's ok for solo-kit considering this function is only used (practically) in event loops.